public/css/module.less

```diff
@@ -299,7 +316,7 @@ ul.bp {
     i {
         float: left;
         font-size: 2.5em;
-        margin-top: -0.1em;
```

Comment:
Please restore this. It's not necessary to change and with this, the icon is at least centered vertically with the title.